checker/src/org/checkerframework/checker/regex/classic/RegexTransfer.java

```diff
@@ -89,6 +90,8 @@ public RegexTransfer(RegexAnalysis analysis) {
                         thenStore.insertValue(firstParam, regexAnnotation);
                     }
                 } catch (FlowExpressionParseException e) {
+                    // report errors here
+                    ((RegexChecker) (analysis.getTypeFactory().getContext())).report(e.getResult(), n);
                     assert false;
```

Comment:
I didn't notice it was just parsing `#1` -- thanks for pointing that out.  I would just ErrorAbort rather than issue a warning and asserting false.